src/plugins/notebook/components/NotebookEntry.vue

```diff
@@ -333,6 +335,17 @@ export default {
                 return found;
             });
 
+            this.timestampAndUpdate();
+        },
+        async timestampAndUpdate() {
+            const user = await this.openmct.user.getCurrentUser();
+
+            if (user !== undefined) {
```

Comment:
I think you want this to be `if (user === undefined)` ?

Reply:
Ah, good catch, thanks.